distribution/ddf-common/src/main/resources/etc/default.policy

```diff
@@ -0,0 +1,79 @@
+priority "grant";
```

Comment:
This whole file now needs to be exported and imported for the migration to work properly. This is to account for changes made by the administrator to allow read access (among other things) to certs that would be placed outside ddf.home.

We also need special permissions added for migration export/import otherwise the functionality will be broken and invalidated in so many place.